llvm/test/CodeGen/AIE/aie2/GlobalISel/inst-select-aie-add-vector-elt-left.mir

```diff
@@ -21,8 +21,8 @@ body:             |
     ; CHECK-LABEL: name: vpush_int32_512
     ; CHECK: [[MOV_RLC_imm10_pseudo:%[0-9]+]]:er = MOV_RLC_imm10_pseudo 12
     ; CHECK-NEXT: [[DEF:%[0-9]+]]:vec512 = IMPLICIT_DEF
-    ; CHECK-NEXT: [[VPUSH_LO_32_:%[0-9]+]]:vec512 = VPUSH_LO_32 [[MOV_RLC_imm10_pseudo]], [[DEF]]
-    ; CHECK-NEXT: PseudoRET implicit $lr, implicit [[VPUSH_LO_32_]]
+    ; CHECK-NEXT: [[VPUSH_HI_32_:%[0-9]+]]:vec512 = VPUSH_HI_32 [[DEF]], [[MOV_RLC_imm10_pseudo]]
+    ; CHECK-NEXT: PseudoRET implicit $lr, implicit [[VPUSH_HI_32_]]
```

Comment:
I don't think there are changes to the instruction selection in this commit. Maybe these tests belong in another commit?